coalib/coala.py

```diff
@@ -22,7 +26,7 @@
 
 def main():
     log_printer = ConsolePrinter()
-    interactor = ConsoleInteractor(log_printer)
+    console_printer = ConsolePrinter()
```

Comment:
In the future we want the possibility to give the user an option of having log_printers which can print to files.